app/src/sharedTest/java/org/oppia/android/app/parser/ListItemLeadingMarginSpanTest.kt

```diff
@@ -485,11 +487,14 @@ class ListItemLeadingMarginSpanTest {
   @RunOn(TestPlatform.ROBOLECTRIC, buildEnvironments = [BuildEnvironment.BAZEL])
   fun testDrawLeadingMargin_forNestedBulletItems_inRtl_isDrawnCorrectlyWithIndentation() {
     val displayLocale = createDisplayLocaleImpl(EGYPT_ARABIC_CONTEXT)
+    val width: Int = context.resources.displayMetrics.widthPixels
 
     val canvas = Canvas()
+    shadowOf(canvas).setWidth(width)
     val paint = Paint()
-    val x = 10
-    val dir = 15
+    val shadowCanvas = shadowOf(canvas)
```

Comment:
Could you move this to right below the canvas line and use it in place of the ``shadowOf(canvas)`` for the ``setWidth`` call above? Ditto below.

Reply:
Done.

Reply:
This is only partly done. I'm suggesting moving the shadowOf about set width such that the code now looks like:

```kotlin
val canvas = Canvas()
val shadowCanvas  shadowOf(canvas)
shadowCanvas.setWidth(width)
val paint = Paint()
...
```

Ditto elsewhere.